login_server/src/cpp/model/gradido/GroupMemberUpdate.cpp

```diff
@@ -67,7 +67,7 @@ namespace model {
 			const static char functionName[] = { "GroupMemberUpdate::validate" };
 			if (mProtoMemberUpdate.user_pubkey().size() != KeyPairEd25519::getPublicKeySize()) {
 				addError(new Error(functionName, "pubkey not set or wrong size"));
-				return TRANSCATION_VALID_INVALID_PUBKEY;
+				return TRANSACTION_VALID_INVALID_PUBKEY;
```

Comment:
Where is the difference?

Reply:
TRANS**CA**TION
TRANS**AC**TION
That's the reason I don't see it at first.

Reply:
^^